### PR TITLE
Changes in response to CodeClimate score for settings_helper.rb

### PR DESCRIPTION
**Pull request by rthbound**
- Breaks method up for clarity and readability

### CODE REVIEW

**Comment by uu59** [comment]
Amazing PR :+1: 
Thanks a lot!!

File: app/helpers/settings_helper.rb [hunk 1]
```diff
@@ -2,42 +2,63 @@ module SettingsHelper
...
+
+    html << "</div>"
+    html.html_safe
+  end
+
+  private
+  def field_resolver(type, html, form, key, opts)
+    case type
     when :hidden
       return form.hidden_field(key)
```
Comment (kou) [inline app/helpers/settings_helper.rb]: Does this code still work? It seems that `field_resolver` should change `html` destructively.

Comment (uu59) [inline app/helpers/settings_helper.rb]: Ah you are right.
I'll fix and add test for that.

Thanks for the catch :star:

Comment (uu59) [inline app/helpers/settings_helper.rb]: Fixed at #130

**Pull request merged**